java-collections-maps-3/src/test/java/com/baeldung/map/invert/InvertHashMapUnitTest.java

```diff
@@ -0,0 +1,60 @@
+package com.baeldung.map.invert;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class InvertHashMapUnitTest {
+
+    Map<String, Integer> sourceMap;
+
+    @BeforeAll
+    void setup() {
+        sourceMap = new HashMap<>();
+        sourceMap.put("Sunday", 0);
+        sourceMap.put("Monday", 1);
+    }
+
+    @Test
+    void test1_invertMapUsingForLoop() {
+        Map<Integer, String> inversedMap = InvertHashMapExample.invertMapUsingForLoop(sourceMap);
+        assertNotNull(inversedMap);
```

Comment:
add a blank line before beginning the assertions (applies to all the tests)

Reply:
Added blank line before beginning the assertions.